test/Microsoft.ML.Core.Tests/UnitTests/TestEntryPoints.cs

```diff
@@ -684,9 +684,7 @@ public void EntryPointCalibrate()
 
             // This tests that the SchemaBindableCalibratedPredictor doesn't get confused if its sub-predictor is already calibrated.
             var fastForest = new FastForestClassification(Env, new FastForestClassification.Arguments());
-            var rmd = RoleMappedData.Create(splitOutput.TrainData[0],
-                RoleMappedSchema.CreatePair(RoleMappedSchema.ColumnRole.Feature, "Features"),
-                RoleMappedSchema.CreatePair(RoleMappedSchema.ColumnRole.Label, "Label"));
+            var rmd = new RoleMappedData(splitOutput.TrainData[0], "Label", "Features");
```

Comment:
>"Features" [](start = 76, length = 10)

DefaultColumnNames?

Reply:
I'm a *little* less excited about this for unit tests.

---
In reply to: [200461236](https://github.com/dotnet/machinelearning/pull/496#discussion_r200461236) [](ancestors = 200461236)

Reply:
it's totally up to you, and probably outside of scope for this PR :)

---
In reply to: [200461719](https://github.com/dotnet/machinelearning/pull/496#discussion_r200461719) [](ancestors = 200461719,200461236)